samples/ClassroomLabs/Modules/Library/Az.LabServices.psm1

```diff
@@ -829,13 +829,13 @@ function New-AzLab {
                 $environmentSettingUri = $labUri + "/environmentsettings/default"
                 $sharedPassword = if ($SharedPasswordEnabled) { "Enabled" } else { "Disabled" }
                 $imageType = if ($image.id -match '/galleryimages/') { 'galleryImageResourceId' } else { 'sharedImageResourceId' }
-                if ($LinuxRdpEnabled) {$linuxRdpState = 'Enabled'} else { $linuxRdpState = 'Disabled' }
-                if ($InstallGpuDriverEnabled) {$gpuDriverState = 'Enabled'} else { $gpuDriverState = 'Disabled' }
+                if ($LinuxRdp) {$linuxRdpState = 'Enabled'} else { $linuxRdpState = 'Disabled' }
+                if ($GpuDriverEnabled) {$gpuDriverState = 'Enabled'} else { $gpuDriverState = 'Disabled' }

```

Comment:
I wonder, since you're renaming anyway - if we should follow the same pattern for LinuxRdp & GpuDriver...  ?   They are used as a switch parameter, so it's implied that passing the switch means "Enabled".

Reply:
I changed the names of these to match the names of the columns currently expected\enforced via the Az.LabServices.BulkOperatations.psm1 script - if we change these, then we'll need to change the names of the corresponding columns passed in via the .csv file.  This feels like more of a breaking change which I don't think is important right now.  We should improve and address this with vNext.  I can log a bug to track this as an issue.